src/Resources/Resources.Test/Models.ResourceGroups/ResourceClientTests.cs

```diff
@@ -281,56 +281,14 @@ public void NewResourceGroupWithoutDeploymentSucceeds()
                 null,
                 new CancellationToken()))
                     .Returns(Task.Factory.StartNew(() => CreateAzureOperationResponse(new ResourceGroup(location: parameters.Location, name: parameters.ResourceGroupName))));
-            SetupListForResourceGroupAsync(parameters.ResourceGroupName, new List<GenericResource>());
+            SetupListForResourceGroupAsync(parameters.ResourceGroupName, new List<GenericResourceExpanded>());
 
             PSResourceGroup result = resourcesClient.CreatePSResourceGroup(parameters);
 
             Assert.Equal(parameters.ResourceGroupName, result.ResourceGroupName);
             Assert.Equal(parameters.Location, result.Location);
         }
 
-        [Fact]
```

Comment:
Why is this not needed?

Reply:
> Why is this not needed?

Error property has been marked as **readonly** for DeploymentValidateResult in [this swagger change](https://github.com/Azure/azure-rest-api-specs/commit/94ac8ea933d3958d3581bdb80783ecb12570a4b3#diff-9d8bbb2e20b057957ef4752ace51cab8R4703). So we could only **Get** Error instead of setting ErrorResponse in this test case.